### PR TITLE
Change the name of some variables.

### PR DESCRIPTION
**Pull request by MaxJPRey**
Change the name of some variables.
Set the value for the "i" in the while loop of the `force_close_desktop()` function.

### CODE REVIEW

File: pyaedt/desktop.py [hunk 1]
```diff
@@ -589,16 +591,16 @@ def _exception(self, ex_value, tb_data):
...
+                design_name = oproject.GetActiveDesign().GetName()
+                if ";" in design_name:
+                    design_name = design_name.split(";")[1]
             except:
-                des_name = ""
+                design_name = ""
         except:
-            proj_name = ""
-            des_name = ""
+            project_name = ""
```
Comment (MaxJPRey) [inline pyaedt/desktop.py:602]: @maxcapodi78 This variable is assigned but never used. We could get rid of it.

File: pyaedt/desktop.py [hunk 2]
```diff
@@ -589,16 +591,16 @@ def _exception(self, ex_value, tb_data):
         """
         try:
             oproject = self._main.oDesktop.GetActiveProject()
-            proj_name = oproject.GetName()
+            project_name = oproject.GetName()
```
Comment (MaxJPRey) [inline pyaedt/desktop.py:594]: @maxcapodi78 This variable is assigned but never used. We could get rid of it.

Comment (maxcapodi78) [reply pyaedt/desktop.py:594]: I agree. that method can be refactored

File: pyaedt/desktop.py (hunk 1 above)
Comment (maxcapodi78) [reply pyaedt/desktop.py:602]: I agree

**Review by akaszynski — APPROVED** [review]
LGTM.

**Pull request merged**